hazelcast/src/main/java/com/hazelcast/map/impl/nearcache/NearCacheProvider.java

```diff
@@ -40,28 +41,28 @@
  */
 public class NearCacheProvider {
```

Comment:
Not necessary at the moment but it would be nice if we could integrate `NearCacheProvider` with `NearCacheManager`.